src/content/protocols/spark.md

```diff
@@ -0,0 +1,414 @@
+| ProxyAdmin                                                              | upgrade                               | Upgrades a contract that has this ProxyAdmin as admin. To date this can only upgrade the `V3InterestRatesFactory`.                                                                                                                                                                                                                                                                                                                                                                               | SubProxy ("Spark Proxy")                                                        |
+| ProxyAdmin                                                              | upgradeAndCall                        | Upgrades a contract that has this ProxyAdmin as admin and then calls a function in it. To date this can only upgrade the `V3InterestRatesFactory`.                                                                                                                                                                                                                                                                                                                                               | SubProxy ("Spark Proxy")                                                        |
+
+## Dependencies
+
+Spark has a core dependency in the Sky Protocol, a stage 0 protocol. This dependency is clear from the fact that all key permissions in Spark contracts are held by the Sky Governance.
+
+In order to bridge funds to other chains, Spark uses Circle's Cross-Chain Transfer Protocol (CCTP) which relies on a set of centralized off-chain signers to provide the bridging attestation. The off-chain signers in the CCTP are not disclosed and we have not found any further specifications on the architecture of the attestation network, emphasizing the centralization risk.
+
+Currently, Chronicle oracles are used in SparkLend with no fallbacks for all asset prices. However, a `KillSwitchOracle` contract is used to set critical prices per asset. Chronicle is a decentralized oracle protocol that computes a median price from multiple sources. The protocol contains validators who push new prices and challengers who can freeze and challenge new prices. The contracts are non-upgradeable.
```

Comment:
discuss stage of decentralization of chronicle itself. Spark inherits chronicle's decentralization stage if Spark does not mitigate centralization risks